src/menu/menu_functions.rs

```diff
@@ -73,7 +73,7 @@ pub fn parse_selection_char(buffer: &str, marker: char) -> ParseResult {
                     return ParseResult {
                         remainder: &buffer[0..index],
                         index: Some(0),
-                        marker: Some(&buffer[index..index + 2]),
+                        marker: Some(&buffer[index..index + 2 * marker.len_utf8()]),
```

Comment:
Well spotted! (Probably rare that a multibyte char is a marker but based on the API we should do that)

Reply:
Yeah, definitely an edge case that isn't common.